docs-ref-conceptual/azure-cli-support-lifecycle.md

```diff
@@ -161,28 +161,27 @@ Azure CLI depends on Python version 3.8 or above. The following table summarizes
 Azure CLI can only be supported on operating systems where the above versions of Python are supported.
 
 * **Windows:** The currently supported versions of Windows client and server meet Python version requirements.
-* **macOS:** The currently supported version of macOS 10.9 and above meet Python version requirements.
+* **macOS:** The currently supported version of macOS 12 and above meet Python version requirements.
 * **Linux:**
   * Each supported operating system has a lifecycle defined by its sponsor organization.
   * Support is typically removed when an operating system goes out of mainline support, at which time we stop testing and supporting it.
   * Here are the supported Linux operating systems for the Azure CLI:
 
     |Operating system|Version|End of support|
     |-|-|-|
-    | Ubuntu| 20.04 LTS| April 2025 |
+    | Ubuntu | 20.04 LTS | April 2025 |
     | | 22.04 LTS | April 2027 |
+    | | 24.04 LTS | June 2029 |
     | Debian | 10 | June 2024 |
     | | 11 | |
     | | 12 | |
     | RHEL | 8 | May 31, 2029 |
     | | 9 | May 31, 2032 |
     | CentOS Stream | 9 | Estimated 2027 |
     | Mariner | 2.0 | |
-    | Alpine | 3.17 | November 22, 2024 |
-    | | 3.16 | May 23, 2024 |
```

Comment:
We cannot remove support for Alpine without notice. Let's discuss internally the process.

Reply:
deletetion removed